app/scripts/views/forms.coffee

```diff
@@ -971,6 +972,26 @@ define [
       @listenToNewFormView()
       @showNewFormViewAnimate()
 
+
+    # Request the history of the form model, the user can click on a revision and see the details
+    # preferably by simply showing the other revision as a model along side in the same views
+    fetchHistory: (formModel) ->
+
+      fielddbHelperModel = new (FieldDB.Datum)(_id: formModel.id)
+      fielddbHelperModel.fetchRevisions().then(((revisions) ->
+        formModel.set 'previous_versions', revisions.map((revisionUrl) ->
+          # TODO can we avoid fetching them until the user clicks on the one they want?
```

Comment:
O, ok I think I see what you're saying now. You want it so that when the user clicks on the history button they aren't bombarded with a (potentially huge) list of fully displayed previous versions, i.e., a full Dative IGT form representation of each version. Instead, clicking on the history button returns a summary of the previous versions and then you can do two-by-two comparisons with full displays of the two versions being compared. That sounds good.

However, are you saying that you think I need to change the OLD API for that? So you want to be able to fetch a summary of an OLD form's `previous_versions` without fetching the entire set of previous versions. Currently that's not possible. It should be an easy add though. (https://github.com/jrwdunham/old/issues/45)

Note that currently you can access any form history by making requests to the `formbackup` resource just like any other resource. I think it can even be searched via the OLD/SQL-ish search pattern. That is, you could retrieve the first n most recent versions of a given form using the search API. I can show you how to do that, if you want.

I still don't think that that gets you what you want though. You want to be able to get a truncated version of all the forms in the history chain, not a subset of the histories.

As for the visuals, I'd like to hold off on that until I get the form views better integrated into (i.e., subclasses of) the "resource" views approach that I've taken with the other resources. That way it should be easier to follow the patterns and get the same kind of UI experience with minimal code rewriting...